README.md

```diff
@@ -119,7 +116,37 @@ The following prerequisites are required:
 
 With your terminal go into the `coatrack/docker` directory where you will find the script `build-and-deploy-images-locally.sh` which builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
 
-
+## Use FIWARE Keyrock for Identity Management instead of Github
+By default, CoatRack uses Github for identity management and login. As an alternative, you can set up CoatRack to use [Keyrock](https://keyrock-fiware.github.io/), which is the FIWARE enabler for Identity Management. In this case you have to set up Keyrock first and afterwards configure CoatRack to work with your Keyrock instance.
+The example set-up explained below is based on the following assumptions: 
+
+* Keyrock version 7.8.1 is deployed via Docker (image `fiware/idm:7.8.1`)
+* Keyrock is accessible via `localhost:3000`
+* CoatRack is accessible via `localhost:8080`
+
+Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
+
+- At first you have to set-up/deploy your Keyrock instance. To do this, just follow the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
+- Please check that Keyrock is running correctly in your local machine by checking if you can access the Keyrock GUI,
```

Comment:
I'd replace the commas at the end of a bullet point by periods.

Reply:
Done